frontend/src/storybook/google-ads/google-ads.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { type Meta, type StoryObj } from '@storybook/react';
+
+import { GoogleAds } from '~/bundles/common/components/google-ads/google-ads.js';
+
+const meta: Meta<typeof GoogleAds> = {
+    component: GoogleAds,
+    title: 'Components/GoogleAds',
+    tags: ['autodocs'],
+};
+
+export default meta;
+type Story = StoryObj<typeof GoogleAds>;
+
+const AddsStory: Story = { args: { children: '' } };
```

Comment:
looks like it should be called Sandbox

each component should have to options
1. Doc
2. Sandbox